### ISSUE TITLE
[A11y] Work Show Page-All - Landmarks

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1999]
### Operating System

Windows

### Browser

Edge

### Affected page

Work Show

### Admin user?

Yes

### Issue Description

https://staging.oregondigital.org/concern/generics/df727084j
https://staging.oregondigital.org/concern/documents/df727057n
https://staging.oregondigital.org/concern/images/1g05fb620

Definite things to look at / fix:

- The breadcrumbs should have a more meaningful label (I'm sure this has been brought up already)
- The "main" landmark shouldn't include the breadcrumbs (also probably already in a ticket)

Maybe:

- The "main" landmark shouldn't include the "CONTACT US" data. I don't know if it's even possible to unravel this, but it's definitely not related to the work itself. (This isn't a major failure, however, as the guidelines for what belongs in main are pretty subjective)
- More generally, the "main" landmark should stop after "DETAILS" and "HOW TO CITE". The rest of the content in these works is sort of tangential to the "main" content. If trivially doable, we could put these in "aside" elements ("complimentary" region) and label them a bit differently so they could indeed stand alone (e.g., instead of "CONTAINED IN THIS OBJECT", perhaps "Contained in Danforth Cooke & Co., Patterson, NJ. Engine purchases by OSN")

That last one is very subjective, and I am just not sure it's worth consideration. I think the work pages (especially compound) may just be trying to do too much work, and that would be a usability issue more than accessibility.

### Suggested Fix

- Give the breadcrumb region a label
- Get breadcrumbs out of the "main" region
- Consider whether it's worth splitting the page up better, then decide it's not and move on

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

Depends on the implementation, but at a minimum validate that the breadcrumbs have a useful region label and aren't part of "main"
